app/Filament/Widgets/RecentDownloadChartWidget.php

```diff
@@ -29,9 +29,9 @@ protected function getData(): array
         $startDate = $this->filters['startDate'] ?? now()->subWeek();
         $endDate = $this->filters['endDate'] ?? now();
 
-        // Convert dates to the correct timezone if necessary
-        $startDate = \Carbon\Carbon::parse($startDate)->startOfDay()->timezone(config('app.timezone'));
-        $endDate = \Carbon\Carbon::parse($endDate)->endOfDay()->timezone(config('app.timezone'));
+        // Convert dates to the correct timezone without resetting the time
+        $startDate = \Carbon\Carbon::parse($startDate)->timezone(config('app.timezone'));
```

Comment:
I like to avoid inline includes where possible can you change this to just `Carbon` and instead include it at the top: `use Carbon\Carbon;`

Reply:
Good point, I changed it :)